src/OpeningHoursAlgorithms.h

```diff
@@ -192,8 +192,7 @@ QVariantList extractOpeningMinutes(const QVariantList& openingHours)
 
 bool isShortAfterMidnight(const QDateTime& dateTime)
 {
-    const int currentHour = dateTime.time().hour();
-    return currentHour >= 0 && currentHour <= 6;
+    return dateTime.time() < QTime(6, 0, 0);
```

Comment:
@smeir I assume this is how we would it like to be? Or do you want to actually treat 6:59 still as the day before?